src/main/java/com/exactpro/th2/sailfish/utils/ProtoToIMessageConverter.java

```diff
@@ -215,6 +231,10 @@ public IMessage fromMetadataFilter(MetadataFilter filter, String messageName) {
         return fromMetadataFilter(filter, FilterSettings.DEFAULT_FILTER, messageName);
     }
 
+    public String getNamespace() {
+        return namespace;
+    }
```

Comment:
Why do you need a getter for the namespace?

Reply:
I have removed dictionaryURI which is used to identify namespace and added internal logic to get namespace from dictionary or default value. This getter returns namespace is used by converter. Honestly this getter is used for loacal test only. Check1 set custom namespace via custom message factory.

Reply:
I think we can make it package-private if it is used only in tests